Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md

```diff
@@ -60,35 +60,41 @@ To verify your policy was assigned correctly:
 
 1. Create a cloud service (classic) in Azure. Get your "Site URL" from Azure portal, this will be your DNS name and CN name for later configuration, for example: `bot.contoso.com`.
 
-1. Set up SSL certificate and upload to the cloud service
+2. Set up SSL certificate and upload to the cloud service
     1. Create a wildcard certificate for your service. This certificate should not be a self-signed certificate. For instance, if your bot is hosted at `bot.contoso.com`, create the certificate for `*.contoso.com`.
     2. Upload the certificate to the cloud service.
     3. Copy the thumbprint for later.
 
-1. Set up cloud service configuration
+3. Set up cloud service configuration
     1. Open powershell, go to the folder that contains file `configure_cloud.ps1`. The file is in the `Samples` directory.
+
     2. Run the powershell script with parameters:
-        * `> .\configure_cloud.ps1 -p .\BetaSamples\LocalMediaSamples\PolicyRecordingBot\ -dns {your DNS name} -cn {your CN name, should be the same as your DNS name} -thumb {your certificate thumbprint} -bid {your bot name} -aid {your bot app id} -as {your bot secret}`, for example `.\configure_cloud.ps1 -p .\BetaSamples\LocalMediaSamples\PolicyRecordingBot\ -dns bot.contoso.com -cn bot.contoso.com -thumb ABC0000000000000000000000000000000000CBA -bid bot -aid 3853f935-2c6f-43d7-859d-6e8f83b519ae -as 123456!@#$%^`
+        `> .\configure_cloud.ps1 -p {path to project} -dns {your DNS name} -cn {your CN name, should be the same as your DNS name} -thumb {your certificate thumbprint} -bid {your bot name} -aid {your bot app id} -as {your bot secret}`
+        
+        For example:
+        
+         `.\configure_cloud.ps1 -p .\V1.0Samples\LocalMediaSamples\PolicyRecordingBot\ -dns bot.contoso.com -cn bot.contoso.com -thumb ABC0000000000000000000000000000000000CBA -bid bot -aid 3853f935-2c6f-43d7-859d-6e8f83b519ae -as 123456!@#$%^`
 
-1. Publish the bot from VS:
+4. Publish the bot from VS:
     1. Right click PolicyRecordingBot, then click `Publish...`. Publish it to the cloud service you created earlier.
 
 ### Test
 
 1. Set up the test meeting and test clients:
    1. Sign in to Teams client with a non-recorded test tenant user.
-   1. Use another Teams client to sign in with the recorded user. (You could use an private browser window at https://teams.microsoft.com)
+   2. Use another Teams client to sign in with the recorded user. (You could use an private browser window at https://teams.microsoft.com). Use Teams client if you couldn't see the call notification by using Teams browser for some reason.
```

Comment:
Maybe rephrase number 2 as
Use another Teams client to sign in with the recorded user. You could use an private browser window and open up https://teams.microsoft.com. If the call notification doesn't appear on web, use the Teams desktop client

Reply:
Good Phrase, I'll take it.